src/index.js

```diff
@@ -7,7 +7,8 @@ var L = require('leaflet'),
 	What3Words = require('./geocoders/what3words'),
 	Google = require('./geocoders/google'),
 	Photon = require('./geocoders/photon'),
-	Mapzen = require('./geocoders/mapzen');
+	Mapzen = require('./geocoders/mapzen'),
+	AcrGis = require('./geocoders/acrgis');
```

Comment:
Looks like a typo here, you're referring the variable `ArcGis` below. Did you build and test the code to check that it works?